code/modules/tgs/v3210/commands.dm

```diff
@@ -42,5 +42,11 @@
 	var/datum/tgs_chat_command/stc = new command_type
 	var/datum/tgs_chat_user/user = new
 	user.friendly_name = sender
+
+	// Discord hack, fix the mention if it's only numbers (fuck you IRC trolls)
+	var/regex/discord_id_regex = regex(@"^[0-9]+$")
```

Comment:
```suggestion
	var/static/regex/discord_id_regex = regex(@"^[0-9]+$")
```
Any reason not to make this static, @Cyberboss ?

Reply:
Very rare, not worth the memory usage